lib/matplotlib/axes/_axes.py

```diff
@@ -6540,6 +6540,11 @@ def hist(self, x, bins=None, range=None, density=False, weights=None,
         --------
         hist2d : 2D histograms
 
+        Notes
+        -----
+        - For large numbers of bins (>1000), 'step' and 'stepfilled' can be
```

Comment:
Please remove the bullet point. There's no need to have a list with just one element here. A plain paragraph is perfectly fine.